pulsar-client-tools/src/main/java/org/apache/pulsar/admin/cli/CmdSchemas.java

```diff
@@ -51,13 +52,24 @@ public CmdSchemas(Supplier<PulsarAdmin> admin) {
         @Parameter(names = { "--version" }, description = "version", required = false)
         private Long version;
 
+        @Parameter(names = {"--all"}, description = "all version", required = false)
```

Comment:
"all versions"?
And maybe it will be better if we can use "--version all" instead of "--all".

Reply:
I'm not sure whether `--version all` is a valid option, maybe `--version-all` or `--all-version`.